pkgs/applications/audio/mopidy/jellyfin.nix

```diff
@@ -2,12 +2,12 @@
 
 python3Packages.buildPythonApplication rec {
   pname = "mopidy-jellyfin";
-  version = "1.0.2";
+  version = "1.0.4";
 
   src = python3Packages.fetchPypi {
     inherit version;
     pname = "Mopidy-Jellyfin";
-    sha256 = "0j7v5xx3c401r5dw1sqm1n2263chjga1d3ml85rg79hjhhhacy75";
+    sha256 = "ny0u6HdOlZCsmIzZuQ1rql+bvHU3xkh8IdwhJVHNH9c=";
```

Comment:
Was there a reason to remove the `sha256-` part of the SRI hash? I'm surprised putting just the base64 part works tbh